platform/measurement_utils.cpp

```diff
@@ -18,24 +17,20 @@
 
 namespace measurement_utils
 {
-using namespace platform;
-using namespace settings;
-using namespace std;
-using namespace strings;
-
-string ToStringPrecision(double d, int pr)
+std::string ToStringPrecision(double d, int pr)
 {
   // We assume that the app will be restarted if a user changes device's locale.
-  static Locale const loc = GetCurrentLocale();
+  static auto const locale = platform::GetCurrentLocale();
 
-  return ToStringPrecisionLocale(loc, d, pr);
+  return ToStringPrecisionLocale(locale, d, pr);
 }
 
-string ToStringPrecisionLocale(Locale loc, double d, int pr)
+std::string ToStringPrecisionLocale(platform::Locale const & loc, double d, int pr)
 {
-  stringstream ss;
-  ss << setprecision(pr) << fixed << d;
-  string out = ss.str();
+  // TODO(AB): Check if snprintf is faster.
```

Comment:
I've made some preliminary tests and `std::snprintf` seems to be much faster than `std::ostringstream`.
I'd keep the current implementation for this PR and launch an additional PR for the `std::snprintf` solution.